platform/ui/tailwind.config.js

```diff
@@ -721,7 +721,7 @@ module.exports = {
     backgroundRepeat: ['responsive'],
     backgroundSize: ['responsive'],
     borderCollapse: ['responsive'],
-    borderColor: ['responsive', 'hover', 'focus', 'active'],
+    borderColor: ['responsive', 'hover', 'focus', 'active', 'group-focus'],
```

Comment:
`group-focus` 👀

Reply:
@dannyrb just clarifying here, `group-hover` or `group-focus` is used for styling a child element when hovering over a parent element. It's a tailwind feature, but some variants like `borderColor` don't have originally group-focus and we have to add this class, like we already have for `textColor`, for example.